src/Compatibility/Core/src/Android/Renderers/PageRenderer.cs

```diff
@@ -221,6 +221,15 @@ void IOrderedTraversalController.UpdateTraversalOrder()
 
 		protected override void OnLayout(bool changed, int l, int t, int r, int b)
 		{
+			var deviceIndependentLeft = Context.FromPixels(l);
+			var deviceIndependentTop = Context.FromPixels(t);
+			var deviceIndependentRight = Context.FromPixels(r);
+			var deviceIndependentBottom = Context.FromPixels(b);
+
+			var destination = Rectangle.FromLTRB(deviceIndependentLeft, deviceIndependentTop,
+				deviceIndependentRight, deviceIndependentBottom);
+
+			(Element as IFrameworkElement)?.Arrange(destination);
```

Comment:
I figure this will get removed once we have more of the window/page parts of MAUI built out